### ISSUE TITLE
Weather Not working on new install

### ISSUE THREAD

**Issue by Don331** [jgyates/genmon#648]
Use the template below if you have an issue or want to report a bug. If you have a question or a feature request you can ignore the questions below. This is a template so feel free to remove any of this text. Try to include the info below when applicable.

***NOTE:*** If you are having issues with your serial connection, please read this page before posting:

https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting

If you are having other issues, please see the following page:

https://github.com/jgyates/genmon/wiki/3.5---General-Troubleshooting

You can search past issues by removing the "is:open" keywords from the search window and adding any keywords that are relevant to your issue.

If you need to send you logs and registers to the developer, if your email is setup and working properly you can click send your logs on the About page in the web interface.

### Expected Behavior
Weather should show up

### Actual Behavior
No weather

### Steps to Reproduce (including precondition)

Setup does show location correctly. I WAS getting the Error in GetObservation: (4711729) : 'OWM' object has no attribute 'weather_at_id' : myweather.py:108
in the logs, but changed the rev to 2.10.0 per another ticket
genmonpi@genmon:/var/log $ pip3 show pyowm
Name: pyowm
Version: 2.10.0
Summary: A Python wrapper around OpenWeatherMap web APIs
Home-page: http://github.com/csparpa/pyowm
Author: Claudio Sparpaglione (@csparpa)
Author-email: csparpa@gmail.com
License: MIT
Location: /usr/local/lib/python3.9/dist-packages
Requires: geojson, requests
Required-by:

### Screenshot or Pictures relating to the problem (if possible)
![2022-01-14_14h54_47](https://user-images.githubusercontent.com/93215357/149584715-2feab06c-0045-44d7-9bd1-6680c2e2b7d3.png)

{Please write here}

### Your Environment

- Generator Model: Generac 18KW
- Genmon Logs: Submit via the About page DONE
- Genmon Version: Version V1.18.04

**Comment by Don331** [comment]
Sorry I was looking in the wrong screen. It's in the Monitor screen. DUH!
